dist/date-range-picker.js

```diff
@@ -1023,7 +1026,6 @@
     var dp = BaseMode(root, emit, opts);
 
     dp.close = noop;
-    dp.destroy = noop;
```

Comment:
I'm curious why this line is being removed? It seems unrelated to the `appendTo` fix, isn't it?

Reply:
nop, suppose not to be deleted, only other change, I will resolve this one, and create new pull request, you may disqualify this one

Reply:
it was removed in this commit 43313b820b989cc4ae8ca29e4de61b4f7947fe92 due to merging